cmd/security-secrets-setup/start_vault.sh

```diff
@@ -27,13 +27,9 @@ listener "tcp" {
               tls_disable = "1" 
               cluster_address = "edgex-vault:8201" 
           } 
-          backend "consul" { 
-              path = "vault/" 
-              address = "edgex-core-consul:8500" 
-              scheme = "http" 
-              redirect_addr = "http://edgex-vault:8200"
-              cluster_addr = "http://edgex-vault:8201"
-          }
+          backend "file" {
```

Comment:
Where does this go once this service is removed?

Reply:
it will go to "bootstrapper" i guess and this one is taking the baby step. so in terms of removing is beyond the scope of this related issue.

Reply:
issue #2882

Reply:
It goes straight into the docker-compose file as static configuration via env var.

Reply:
Should also check to see if the snap depends on this script to generate the config file, or uses something else.

Reply:
@lenny-intel please suggest a change. i do not what you want me to do.